packages/runtime/container-runtime-definitions/src/snapshot.ts

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ISnapshotTree } from "@fluidframework/protocol-definitions";
+
+export type PropertyValues<T> = T[keyof T];
+
+export const containerSnapshotFormatVersions = {
+    missing: undefined,
+    next: "0.1",
```

Comment:
I do not see containerSnapshotFormatVersions.next used anywhere in he code.

Reply:
It isn't used, but we do check for undefined and treat that else scenario specially.